report/src/conclusion/conclusion.tex

```diff
@@ -0,0 +1,13 @@
+\chapter{Conclusion}
+This report examines a stakeholder's problems regarding the management of meetings. 
+By performing a usability evaluation of a developed prototype, it is concluded that a solution to their problem might involve a presence detection system to establish whether the stakeholder's meeting rooms are occupied. 
+Chapter \ref{chap:presence_intro} describes and evaluates technologies appropriate for presence detection, guided by a set of criteria considered necessary to address the stakeholder's problem. 
+Of these technologies, Bluetooth Low Energy was found to be the best candidate. 
+Three techniques for presence detection using BLE were presented and analyzed in Chapter \ref{sec:presence_detection_using_ble}. 
+It was found that fingerprinting using k nearest-neighbors would be a suitable candidate for presence detection.
```

Comment:
```suggestion
Through an exploration of technologies suitable for presence detection, an extensive evaluation was conducted to address the stakeholder's problem (see Chapter \ref{chap:presence_intro}). 
After careful consideration, Bluetooth Low Energy (BLE) emerged as the most promising choice. 
Building upon this foundation, Chapter \ref{sec:presence_detection_using_ble} delves into three distinct techniques for presence detection utilizing BLE.
Following a detailed assessment, the fingerprinting technique, using the k-nearest neighbors approach, emerged as a suitable candidate for presence detection.```
I don't like the way this is phrased. It feels like a dull walkthrough and doesn't flow well. I suggest we fix this by adding some coherence - see my suggestion - using words such as "following", "building upon", and so on.

Reply:
I don't like the way this is phrased. It feels like a dull walkthrough and doesn't flow well. I suggest we fix this by adding some coherence - see my suggestion - using words such as "following", "building upon", and so on.